kythe/typescript/indexer.ts

```diff
@@ -141,13 +141,15 @@ export enum TSNamespace {
 }
 
 /**
- * Context represents the environment a node is declared in, and only applies to
- * nodes with multiple declarations. The context may be used for disambiguating
- * node declarations. A Getter context means the node is declared as a getter; a
- * Setter context means it is declared as a setter.
+ * Context represents the environment a node is declared in, and may be used for
+ * disambiguating a node's declarations if it has multiple.
  */
 export enum Context {
+  /** No disambiguation about a node's declarations. */
```

Comment:
This is lazy, but that's not mentioned here. Could you include `@see SymbolVNameStore` here?

Reply:
Done.